site/gdocs/OwidGdoc.tsx

```diff
@@ -57,7 +57,7 @@ export function OwidGdoc({
         content.title
     }". Published online at OurWorldInData.org. Retrieved from: '${`${BAKED_BASE_URL}/${slug}`}' [Online Resource]`
 
-    const bibtex = `@article{owid${slug.replace(/-/g, "")},
+    const bibtex = `@article{owid${slug.replace(/[-\/]/g, "")},
```

Comment:
nice! as I said on slack, it would be nice to whitelist dashes here, since they make the entry name much more readable.